test/e2e/specs-gutenberg/wp-calypso-gutenberg-focused-launch-spec.js

```diff
@@ -262,6 +262,24 @@ describe( `[${ host }] Calypso Gutenberg Editor: Focused launch on (${ screenSiz
 			assert( isFocusedLaunchModalPresent, 'Focused launch modal did not open.' );
 		} );
 
+		step( 'Can persist previously selected domain in focused launch', async function () {
+			const selectedDomainSuggestionItemNameSelector = By.css(
+				'.domain-picker__suggestion-item.is-selected .domain-picker__suggestion-item-name'
+			);
```

Comment:
Isn't it possible to select the selected domain suggestion item by text (i.e. use the helper)? That would resemble the user's behavior a bit better.

Reply:
I did the relevant changes here and it is passing. https://github.com/Automattic/wp-calypso/pull/51847/commits/5563de8b6a2a3ed4c902de5360203af1d7bbee7e